### ISSUE TITLE
IMG_SHORTCUTS_DEFAULT has no default

### ISSUE THREAD

**Issue by busche** [davidB/gradle-getdown-plugin#24]
When setting up a new project as documented on the Github page (extending buildscript dependencies, applying the plugin, configuring the getdown part ...) and running the assembleApp task, I get the following error

:makeIcons FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':makeIcons'.
> Could not find property 'IMG_SHORTCUTS_DEFAULT' on task ':makeIcons'.

My environment:
(embedded Gradle in Eclipse Build id: 20160218-0600)
Gradle Version: 2.12
Java Home: C:\Program Files\Java\jdk1.8.0_66
JVM Arguments: None
Program Arguments: None
Gradle Tasks: assembleApp

The documentation does not state anything for this property. What shall I do / configure?

**Comment by busche** [comment]
OK, the bug should be located here:
https://github.com/davidB/gradle-getdown-plugin/blob/ef55713d18a6464355a5e6e767fa4530731f6af9/src/main/groovy/bundles/GetdownPlugin.groovy#L70
use absolute referencing to IMG_SHORTCUTS_DEFAULT

**Comment by busche** [comment]
workaround:

- create  folder src/dist (if not yet existing)
- create file shortcut-<n>.png (with <n> 16,32,64,128 - unsure if 256 also works) therein
- add shortcuts = ['shortcut-64.png'] as a configuration parameter in the gradle getdown { } snippet

**Comment by davidB** [comment]
Thanks for reporting, I'll update the doc and improve it (in the new renamed plugin).

**Comment by lwouis** [comment]
Any news on this? I'm getting the same error today on a fresh setup. I don't know Groovy so I have no clue what IMG_SHORTCUTS_DEFAULT is, and what I could do to fix this.

**Comment by Ali-RS** [comment]
David I am getting the same error.

**Comment by busche** [comment]
Well, I've posted the workaround above, which works.
Maybe @davidB gives a pointer to the renamed plugin or changes it here.
I am not using the plugin currently, and since the shortcut depends on your software, it is best to fix it locally at your place.

**Comment by Ali-RS** [comment]
> Well, I've posted the workaround above, which works.

Yes, @busche i already fixed it locally  with your workaround. Thanks.
